scripts/generate_dump

```diff
@@ -1157,6 +1157,67 @@ collect_mellanox_dfw_dumps() {
     done
 }
 
+###############################################################################
+# Runs a given marvellcmd command in all namesapces in case of multi ASIC platform
+# Globals:
+#  NUM_ASICS
+# Arguments:
+#  cmd: The command to run. Make sure that arguments with spaces have quotes
+#  filename: the filename to save the output as in $BASE/dump
+#  do_gzip: (OPTIONAL) true or false. Should the output be gzipped
+# Returns:
+#  None
+###############################################################################
+save_marvellcmd() {
+    trap 'handle_error $? $LINENO' ERR
+
+    mkdir -p $LOGDIR/sdkdump
```

Comment:
>sdkdump [](http://example.com/codeflow?start=21&length=7)

Do you want to clean up this folder if there are old files, before you save new file?

Reply:
Full path of the sdkdump will be something like below, where parenet directory(sonic_dump_sonic_XXXXXXX_XXXX) is always new each time we execute "show techsupport" command. So old file will not be present.
"/var/dump/sonic_dump_sonic_20230615_125930/dump/sdkdump/".